examples/raytrace-parallel/README.md

```diff
@@ -12,4 +12,4 @@ You can build the example locally with:
 $ python3 run.py
 ```
 
-and then visiting http://localhost:8080 in a browser should run the example!
+and then visiting http://localhost:8000 in a browser should run the example!
```

Comment:
`wasm-audio-worklet` uses port 8080, but `raytrace-parallel` uses port 8000:

https://github.com/rustwasm/wasm-bindgen/blob/792be2d1faf7072fd51358558eacfc58b6c48642/examples/raytrace-parallel/server.py#L12

Reply:
That's fine, thank you!